features/create_membership_application.feature

```diff
@@ -4,6 +4,12 @@ Feature: As a visitor
 
   PT Feature: https://www.pivotaltracker.com/story/show/133940725
```

Comment:
Could you make this on line four and just PT: link ? 
The second feature gets coloured in some ide(?)'s and that could be confusing.